### ISSUE TITLE
Segmentation fault: 11

### ISSUE THREAD

**Issue by tonycoco** [adopted-ember-addons/ember-cli-sass#1]
After including node-neat and when using this...

``` js
var app = new EmberApp({
  sassOptions: {
    includePaths: require('node-neat').includePaths
  }
});
```

I get seg faults when trying to access these from my scss file...

``` scss
@import "bourbon/bourbon";
@import "neat-helpers";
@import "neat/neat";
```

```
Serving on http://0.0.0.0:4200
Segmentation fault: 11
```

Any trick to getting this to work with the CLI?

**Comment by jrjohnson** [comment]
We're getting this error again: https://travis-ci.org/ilios/frontend/builds/49666398

**Comment by aexmachina** [comment]
This is definitely an issue with node-sass, ember-cli-sass is a thin wrapper around node-sass (via broccoli-sass - also a thin wrapper).

**Comment by jongold** [comment]
I'm getting this with `3.1.0`.

```
$ ember serve
version: 0.2.0
Livereload server on port 35729
Serving on http://localhost:4200/

zsh: segmentation fault  ember s
```

The app compiles fine (apart from the scss, obviously) if I take `ember-cli-sass` out of my `package.json`. I guess this is related to the node-sass bug, but I thought using `3.1.0` would fix it.

Have tried `rm -rf node_modules` folder, `npm cache clean`, switching versions of node (`0.10.30` & `0.12.0`) and nothing.

Maybe my package.json file is useful? Just picking up this project again after it being dormant for a few months - upgrading from `ember-cli@0.0.40` to `0.2.0`, so a bunch of things have just been updated, but this is the only thing that seems broken.
```json
{
  "name": "paz-ember",
  "version": "0.0.0",
  "description": "Small description for paz-ember goes here",
  "private": true,
  "directories": {
    "doc": "doc",
    "test": "tests"
  },
  "scripts": {
    "start": "ember server",
    "lint": "jshint app config",
    "build": "ember build",
    "test": "ember test"
  },
  "repository": "",
  "engines": {
    "node": ">= 0.10.0"
  },
  "author": "",
  "license": "MIT",
  "devDependencies": {
    "broccoli-asset-rev": "^2.0.0",
    "ember-cli": "^0.2.0",
    "ember-cli-app-version": "0.3.2",
    "ember-cli-babel": "^4.0.0",
    "ember-cli-dependency-checker": "0.0.8",
    "ember-cli-htmlbars": "0.7.4",
    "ember-cli-ic-ajax": "0.1.1",
    "ember-cli-inject-live-reload": "^1.3.0",
    "ember-cli-qunit": "0.3.9",
    "ember-cli-sass": "3.1.0",
    "ember-cli-uglify": "1.0.1",
    "ember-data": "1.0.0-beta.15",
    "ember-export-application-global": "^1.0.2",
    "ember-validations": "^2.0.0-alpha.3",
    "express": "^4.8.5",
    "glob": "^4.4.2",
    "precommit-hook": "^1.0.7"
  },
  "precommit": [
    "lint"
  ]
}
```
:sos:

**Comment by jongold** [comment]
More digging, looks like the node-sass bug was reintroduced in 2.0.1 which broccoli-sass is using. Will see if I can override it later.

**Comment by darioghilardi** [comment]
I still have this issue, right after installing `ember-cli-sass`. Any workaround or solution?

**Comment by aexmachina** [comment]
@darioghilardi this is certainly going to be a node-sass issue, have you tried using node-sass@3.0.0-beta.7? You should be able to get this by reinstalling ember-cli-sass.

**Comment by joshuairl** [comment]
Cannot get past this error and if I switch things around I'm getting the infamous `Bus error: 10`
I'm running latest io.js and `brew install libsass --HEAD`

**Comment by aexmachina** [comment]
@joshuairl, there's really nothing ember-cli-sass can do about this - it's a problem with node-sass. Here's what I'd do:

- `npm install --save ember-cli-sass@3.3.1`
- If that doesn't work then try `rm -rf node_modules && npm cache clean && npm install`
- If you're still having problems then post your issue on node-sass

Good luck!

**Comment by mlunoe** [comment]
I had the same issue in my ember project (with yarn workspaces - not sure if that is relevant to the issue).

I tried clearing out `tmp` and `node_modules` folders, but was still receiving the SIGSEGV error.
Running the build with the `DEBUG=*` flag I found that it died right after compiling certain files. If I deleted the whole folder, where it was dying (containing scss files that were doing `@import`s) and keeping other folders with similar scss files, it would build.

I was never able to debug the libsass process itself (so deeply nested in the node_modules), but after cloning a new instance of my repository (moving over my `.git`-folder) everything worked again.

I hope this helps others in a similar situation!
